modules/tinymce/src/core/test/ts/browser/focus/HighlightOnFocusTest.ts

```diff
@@ -54,6 +60,54 @@ describe('browser.tinymce.core.focus.HighlightOnFocus', () => {
       await focusOutsideEditor();
       assertIsNotHighlighted(editor);
     });
+
```

Comment:
Should add some tests for navigating to the toolbar/menubar/statusbar via keyboard shortcuts. Also, the format of your tests are all the same, so this could be separated into a new function. Just a suggestion though.
```suggestion
    const pAssertHighlightOnFocus = async (action: () => void, wait: async () => void), exit: () => void) => {
      const editor = hook.editor();
      assertIsNotHighlighted(editor);
      editor.focus();
      assertIsHighlighted(editor);
      action();
      await wait();
      await pAssertIsNotHighlighted(editor);
      exit();
      assertIsHighlighted(editor);
    };
```

Reply:
Refactored. Thanks!